client/js/render.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
+const $ = require("jquery");
+const templates = require("../views");
+const options = require("./options");
+const utils = require("./utils");
+const sorting = require("./sorting");
+
+const chat = $("#chat");
+const sidebar = $("#sidebar, #footer");
```

Comment:
Probably not? I'm not sure why it was in it in `lounge.js` so I'll need to figure that out in the first place before I delete the footer.